vision/cloud-client/detect/vision_batch_annotate_files.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_batch_annotate_files]
+
+from google.cloud import vision_v1
+from google.cloud.vision_v1 import enums
+import io
+
+
+def sample_batch_annotate_files(file_path="resources/kafka.pdf"):
+    """
+    Perform batch file annotation
+
+    Args:
+      file_path Path to local pdf file, e.g. /path/document.pdf
```

Comment:
Should this be dropped since there is now a default above?

Reply:
Even with defaults, it's still good style to have docstrings for all sorts of reasons, like bubbling up in error messages or providing clear context. Or are we going for the bare minimum inside docs?

Reply:
In the past our snippets on docs go for the bare minimum, but that's usually because the region tags didn't include the doc strings and now with it outside the function def maybe it should have them. 

I'm also waiting on the new style guide rules for Python.
So... Idk! 

This default in the function def might be more helpful `path/to/your/document.pdf` otherwise, we'll get folks saying they don't have that file.

Reply:
@kurtisvg docstrings or no?

That file is included in the resources directory, which is the point of the default. Do you think that will still be problematic for some users?

Reply:
I lean slightly towards no (or minimal) docstring because it often feels too verbose. A single line explaining the action the snippet takes seems the most valuable.

RE: the file in the resources directory - the goal is to optimize for folks copy-pasting from c.g.c, which means they would need to supply their own file. With that in mind `path/to/your/document.pdf` makes the most sense because it's more obvious they need to change it.